libcomposefs/lcfs-writer-erofs.c

```diff
@@ -500,6 +498,18 @@ static uint32_t compute_erofs_chunk_bitsize(struct lcfs_node_s *node)
 	return chunkbits;
 }
 
+/* In composefs for non-inline files we just have a "stub" block filled
+ * with a small amount of data, because the real data lives in the underlying
+ * filesystem. The chunks are filled with 0xFF.
+ */
```

Comment:
This comment isn't quite right. what happens is that we want to make the file look like it has the right size, but filled with zeros. But we want to do this in a "hole" way, i.e. without storing a bunch of zeros. In the chunked format the file data is a list of offsets to blocks of a specific size (the chunk format). We chose the maximum size for the chunk format that is smaller than the file we need (so typically we'll only use one chunk). But then, instead of using a real pointer we use the EROFS_NULL_ADDR which erofs uses for holes.

Reply:
(ie. EROFS_NULL_ADDR makes erofs produce a bunch of zeros for that block)

Reply:
Thanks! I'll fix that one up in a followup.